lib/rules/prefer-rest-params.js

```diff
@@ -32,6 +32,28 @@ function getVariableOfArguments(scope) {
     return null;
 }
 
+/**
+ * Checks if the given reference is not normal member access.
+ *
+ * - arguments         .... true    // not member access
+ * - arguments[i]      .... true    // computed member access
+ * - arguments[0]      .... true    // computed member access
+ * - arguments.length  .... false   // normal member access
```

Comment:
What about `arguments["length"]` (with a string literal)? Should we relax the rule in that case?